onnxconverter_common/container.py

```diff
@@ -95,7 +95,7 @@ def __init__(self, target_opset):
         # ONNX nodes (type: NodeProto) used to define computation structure
         self.nodes = []
         # ONNX operators' domain-version pair set. They will be added into opset_import field in the final ONNX model.
-        self.node_domain_version_pair_sets = set()
+        self.node_domain_version_pair_sets = {('', target_opset)}
```

Comment:
The converter choose the minimal opset version on the final step. will this change break the existing behavior?

Reply:
made new change about it.